### PR TITLE
Depend on 10.0.3 or above

### PR DESCRIPTION
**Pull request by karakayasemi**
app depends on min 10.0.3. See #8

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
:+1:

File: appinfo/info.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <version>0.0.1</version>
     <namespace>Security</namespace>
     <dependencies>
-        <owncloud min-version="9.0" max-version="10.2" />
+        <owncloud min-version="10.0.3" max-version="10.0.99" />
```
Comment (DeepDiver1975) [inline appinfo/info.xml]: leave my to 10.0

**Comment by karakayasemi** [comment]
By the way, I have been stole this 10.0.99 syntax from user_ldap :)

**Comment by DeepDiver1975** [comment]
> By the way, I have been stolen this 10.0.99 syntax from user_ldap :)

well - wrong there as well ;-)

**Pull request merged**